app/forms/steps/capital/property_form.rb

```diff
@@ -19,6 +19,13 @@ class PropertyForm < Steps::BaseFormObject
       validates :has_other_owners, inclusion: { in: YesNoAnswer.values }
       validates :percentage_partner_owned, presence: true, if: :include_partner?
 
+      validates_numericality_of :percentage_applicant_owned, greater_than_or_equal_to: 0.0,
```

Comment:
Wondering if this should be just greater than 0 - else they don't really own it at all, right?

Reply:
Ignore, the partner could own 100%